modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -73,13 +73,20 @@ class Message extends Component<Renderer.Message> {
       'store'
     ])
 
+    const childIsSessionReset = wrapped.type === 'session_reset'
+    const child = wrapped && (
+      <Message {...sanitizedProps} keyboard={Keyboard} noBubble={!childIsSessionReset} payload={wrapped} />
+    )
+    if (childIsSessionReset) {
+      return child
+    }
+
     const props = {
       ...sanitizedProps,
       ...messageDataProps,
       keyboard: Keyboard,
-      children: wrapped && <Message {...sanitizedProps} keyboard={Keyboard} noBubble={true} payload={wrapped} />
+      children: child
```

Comment:
This is handling only one case of different css for that particular message type. The wrapper should handle applying styling for elements it wraps around, and not handled by the custom message type. 

If the message is wrapped twice, this will not work. Instead, you could apply the CSS class ".bpw-bubble-session_reset" in the Wrapper, if the child type is session_reset.